docs/styling-engines.md

```diff
@@ -0,0 +1,27 @@
+# 🕴️ Styling engines
+
+### Using UIToolkit in external windows 🪟
+UIToolkit's `Dropdown` and `Tooltip` components have libraries (`popperjs` and `react-select`) that inject styling with `@emotion`. If you try using these components in a a window outside the main one (e.g. popout) the styling will only be applied to the main document.
+
+So if you are using UIToolkit in external windows you need to wrap it in a `CacheProvider` with the container argument set to the document where the styling will be implemented. For example:
+
+```
+import createCache from '@emotion/cache';
+import { CacheProvider } from '@emotion/react';
+
+const emotionCache = createCache({
+    container: popoutDocument.head
```

Comment:
The 'key' is missing in the example (See https://emotion.sh/docs/@emotion/cache)

Reply:
👍 will add.